### ISSUE TITLE
Raise TypeError when status is None

### ISSUE THREAD

**Issue by Aculeasis** [rohankapoorcom/zm-py#51]
``` python
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 446, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 641, in async_device_update
    raise exc
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/homeassistant/homeassistant/components/zoneminder/camera.py", line 57, in update
    self._is_recording = self._monitor.is_recording
  File "/usr/local/lib/python3.9/site-packages/zoneminder/monitor.py", line 169, in is_recording
    return int(status) == STATE_ALARM
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

**Comment by nabbi** [comment]
PR #49 and #50 have proposed changes related to this bug.
I added to the confusion by submitting #56 with yet another variation which should cover these unknown scenarios in which we are getting a non Integer value

**Comment by nabbi** [comment]
Also. #48 appears to be the same issue.

**Comment by nabbi** [comment]
fixed in v0.5.3
